### PR TITLE
use some user data to fill EP form

### PR DESCRIPTION
**Pull request by dreamfall**
updated a couple of EP fields

### CODE REVIEW

File: app/controllers/form_controller.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ class FormController < ApplicationController
   before_action :check_trade_count_limit, only: :new_international_trade_form
 
   def new_innovation_form
-    form_answer = FormAnswer.create!(user: current_user, account: current_user.account, award_type: 'innovation', document: {
+    form_answer = FormAnswer.create!(user: current_user, account: current_user.account, award_type: "innovation", document: {
```
Comment (matthewford) [inline app/controllers/form_controller.rb]: Line is too long. [125/100]

File: app/controllers/form_controller.rb [hunk 2]
```diff
@@ -25,26 +25,33 @@ def new_innovation_form
   end
 
   def new_international_trade_form
-    form_answer = FormAnswer.create!(user: current_user, account: current_user.account, award_type: 'trade')
+    form_answer = FormAnswer.create!(user: current_user, account: current_user.account, award_type: "trade")
```
Comment (matthewford) [inline app/controllers/form_controller.rb]: Line is too long. [108/100]

File: app/controllers/form_controller.rb [hunk 3]
```diff
@@ -25,26 +25,33 @@ def new_innovation_form
...
     redirect_to edit_form_url(form_answer)
   end
 
   def new_sustainable_development_form
-    form_answer = FormAnswer.create!(user: current_user, account: current_user.account, award_type: 'development')
+    form_answer = FormAnswer.create!(user: current_user, account: current_user.account, award_type: "development")
```
Comment (matthewford) [inline app/controllers/form_controller.rb]: Line is too long. [114/100]

File: app/controllers/form_controller.rb [hunk 4]
```diff
@@ -25,26 +25,33 @@ def new_innovation_form
...
     redirect_to edit_form_url(form_answer)
   end
 
   def new_enterprise_promotion_form
-    form_answer = FormAnswer.create!(user: current_user, account: current_user.account, award_type: 'promotion')
+    form_answer = FormAnswer.create!(user: current_user, account: current_user.account, award_type: "promotion", document: {
```
Comment (matthewford) [inline app/controllers/form_controller.rb]: Line is too long. [124/100]

File: app/controllers/form_controller.rb [hunk 5]
```diff
@@ -25,26 +25,33 @@ def new_innovation_form
...
+      email: current_user.email,
```
Comment (matthewford) [inline app/controllers/form_controller.rb]: Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.

File: app/controllers/form_controller.rb [hunk 6]
```diff
@@ -25,26 +25,33 @@ def new_innovation_form
...
+      first_name: current_user.first_name,
+      last_name: current_user.last_name,
+      phone: current_user.phone_number,
+      title: current_user.title
+    })
```
Comment (matthewford) [inline app/controllers/form_controller.rb]: Indent the right brace the same as the first position after the preceding left parenthesis.

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step3.rb [hunk 7]
```diff
@@ -3,24 +3,34 @@ class QAE2014Forms
   class << self
     def promotion_step3
       @promotion_step3 ||= proc do
-        textarea :reasons_to_nominate, "Explain why you think your nominee deserves this award." do
+        textarea :reasons_to_nominate, "Explain why your nominee deserves a Queen's Award for Enterprise Promotion." do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step3.rb]: Line is too long. [119/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step3.rb [hunk 8]
```diff
@@ -3,24 +3,34 @@ class QAE2014Forms
...
           ref "C 1"
           required
           context %(
             <p>
-              Include details of how they have made a significant contribution in their area of activity whether in public/private and/or voluntary sectors. Give as much detail as possible about what makes your nominee stand out against others.
+              How have they made a significant contribution in their area of activity whether in public/private and/or voluntary sectors?
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step3.rb]: Line is too long. [137/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step3.rb [hunk 9]
```diff
@@ -3,24 +3,34 @@ class QAE2014Forms
...
+              What makes your nominee stand out against others?
             </p>
                     )
           rows 3
           words_max 500
         end
 
-        textarea :benefits_from_nominee_service, "Describe the benefits resulting from the nominee’s services to a particular field, group, community or society as a whole. " do
+        textarea :benefits_from_nominee_service, "Describe the benefits of your nominee’s services to a particular field, group, community or society as a whole." do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step3.rb]: Line is too long. [165/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step3.rb [hunk 10]
```diff
@@ -34,13 +44,13 @@ def promotion_step3
           words_max 300
         end
 
-        options :nominee_deservers_lifetime_achievement_award, "Do you think your nominee deserves a Lifetime Achievement Award?" do
+        options :nominee_deservers_lifetime_achievement_award, "Your nominee is eligible for a Lifetime Achievement Award?" do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step3.rb]: Line is too long. [126/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step3.rb [hunk 11]
```diff
@@ -34,13 +44,13 @@ def promotion_step3
...
           required
           ref "C 4"
           yes_no
           context %(
             <p>
-              A Lifetime Achievement Award is a special Enterprise Promotion Award that is given for outstanding, consistent and effective promotion of business enterprise skills and attitudes <u>over at least ten years.</u>
+              A Lifetime Achievement Award is a special Enterprise Promotion award that is given for outstanding, consistent and effective promotion of business enterprise skills and attitudes <u>over at least ten years.</u>. All those nominated for Lifetime Achievement are still considered for the standard award.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step3.rb]: Line is too long. [315/100]

**Comment by RuslanHamidullin** [comment]
Look ok for me :+1:

File: app/controllers/form_controller.rb [hunk 12]
```diff
@@ -13,38 +13,63 @@ class FormController < ApplicationController
...
+      award_type: "innovation",
+      document: {
         company_name: current_user.company_name,
         principal_address_building: current_user.company_address_first,
         principal_address_street: current_user.company_address_second,
         principal_address_city: current_user.company_city,
         principal_address__country: current_user.company_country,
         principal_address_postcode: current_user.company_postcode
-      })
+    })
```
Comment (matthewford) [inline app/controllers/form_controller.rb]: Indent the right brace the same as the start of the line where the left brace is.

File: app/controllers/form_controller.rb [hunk 13]
```diff
@@ -13,38 +13,63 @@ class FormController < ApplicationController
...
+      user: current_user,
+      account: current_user.account,
+      award_type: "promotion",
+      document: {
+        email: current_user.email,
+        first_name: current_user.first_name,
+        last_name: current_user.last_name,
+        phone: current_user.phone_number,
+        title: current_user.title
+    })
```
Comment (matthewford) [inline app/controllers/form_controller.rb]: Indent the right brace the same as the start of the line where the left brace is.

**Pull request merged**